staging/src/kubevirt.io/api/core/v1/schema.go

```diff
@@ -569,6 +569,9 @@ type Disk struct {
 	// If specified, the virtual disk will be presented with the given block sizes.
 	// +optional
 	BlockSize *BlockSize `json:"blockSize,omitempty"`
+	// If specified the disk is made sharable and multiple write from different VMs are permitted
+	// +optional
+	Shareable bool `json:"shareable,omitempty"`
```

Comment:
should this be `*bool` like `DedicatedIOThread`?

Reply:
if I use a bool, it relies on the golang default and is false if not set. With the pointer, I need to check if it isn't `nil` or not and then set the value. I can do it!

Reply:
yeah suggesting just for consistency sake